.babelrc

```diff
@@ -1,3 +1,3 @@
 {
-  "presets": ["es2015-node5"]
+  "presets": ["@babel/env"]
```

Comment:
Could we use sshd here? Your solution is ace but I wonder if we can leverage something off the shelf.